hugegraph-core/src/main/java/com/baidu/hugegraph/auth/StandardAuthManager.java

```diff
@@ -59,6 +60,8 @@ public StandardAuthManager(HugeGraphParams graph) {
         this.graph = graph;
         this.eventListener = this.listenChanges();
         this.usersCache = this.cache("users");
+        this.validateCache = CacheManager.instance().cache("validate_pwd");
```

Comment:
this.cache("users_pwd")

Reply:
`cache(string)`  returns `Cache<Id, HugeUser>`, but we need `Cache<Id, String>` now